PowerNSX.psm1

```diff
@@ -6105,6 +6105,9 @@ function New-NsxController {
 
     The New-NsxController cmdlet deploys a new NSX Controller.
 
+    Note that a password is still required as a parameter for controller 2 and 3.
```

Comment:
Id prefer to fix this issue (make the password not accepted for anything other than first controller) rather than doc the wrong behaviour.  Up to it?